src/api/client.js

```diff
@@ -28,3 +29,16 @@ export function getAssets(courseId, { page = 0, pageSize = 50, sort = 'sort', as
     },
   );
 }
+
+export function deleteAsset(courseId, assetId) {
+  return fetch(
+    `${endpoints.assets}/${courseId}/${assetId}`, {
+      credentials: 'same-origin',
+      method: 'delete',
+      headers: {
+        Accept: 'application/json',
+        'X-CSRFToken': Cookies.get('csrftoken'),
```

Comment:
TODO: Ari suggested having a top-level request object that has common attributes that other requests can extend, so we don't have to add common headers to each request.

Reply:
I have added it to my fancy TODO chart.